simulations/vip-130/testnet/simulations.ts

```diff
@@ -0,0 +1,6 @@
+import { forking, testVip } from "../../../src/vip-framework";
+import { vip130Testnet } from "../../../vips/vip-130/testnet";
+
+forking(30855643, () => {
+  testVip("VIP-130-testnet Swap routers accept ownership", vip130Testnet());
+});
```

Comment:
I would add checks after the execution of the VIP, to verify who is the new owner

Reply:
Added, 7e4cd21462d18171f65759acdda628511d6949d6